soh/soh/Enhancements/debugger/ImGuiHelpers.cpp

```diff
@@ -19,3 +19,13 @@ void InsertHelpHoverText(const std::string& text) {
         ImGui::EndTooltip();
     }
 }
+
+void PaddedSeparator(bool topPadding, bool bottomPadding, float x, float y) {
```

Comment:
not the biggest fan of these names, since `topPadding` and `bottomPadding` are bools, it might make sense to name them `padTop` and `padBottom` (since they aren't defining *how much* padding goes on the top/bottom)

as for `x` and `y`, it's just not clear at all what those do

also, can you think of any times when someone would want to have `x` and `y` be different for the top vs bottom padding?

Reply:
The idea was to be able to pad only the top when the seperator is at the end of a list of options, and the other way around for when it's at the top of the options. Or for example, if you have a seperator just below a heading with stuff underneath, you might want to not have the top padding or a little top padding, and more bottom padding. Just figured I'd keep it as flexible as possible, also for possibly using this same one for the F1 menu in the future (which I kinda want to).

That said, given that it's a horizontal seperator, `x` is probably never really going to be useful, so that can just be removed entirely.

On the naming part, I agree. How about `padTop`, `padBottom' (as you said) and `verticalPadding` in place of `y`?

Reply:
> `verticalPadding` in place of `y`?

that's still a bit unclear to me, considering there will still be padding when `verticalPadding` is set to `0`

Reply:
maybe "additionalVerticalPadding`?

Reply:
I'll make it `extraVerticalPadding` then to make it slightly shorter.